lib/publishing_api_finder_publisher.rb

```diff
@@ -100,7 +100,7 @@ def details
     details.merge(
       "subscription_list_title_prefix" => details.fetch("subscription_list_title_prefix", {}),
       "email_filter_facets" => email_filter_facets,
-    ).except("document_noun", "facets", "filter", "generic_description", "reject", "summary")
+    ).except("canonical_link", "document_noun", "facets", "filter", "generic_description", "reject", "summary")
```

Comment:
I feel like, with a blacklist here, this is just going to keep happening.  But maintaining a whitelist could be just as painful...

Reply:
I agree. I will add a tech-debt card to look at this in more detail :+1: